mitmproxy/flowfilter.py

```diff
@@ -432,7 +434,7 @@ def __call__(self, f):
         if not f.client_conn or not f.client_conn.peername:
             return False
         r = f"{f.client_conn.peername[0]}:{f.client_conn.peername[1]}"
-        return f.client_conn.peername and self.re.search(r)
```

Comment:
is there a reason why you removed this? Just wondering

Reply:
That check is already being made three lines up. :)

Reply:
oh true 😅